circle.yml

```diff
@@ -585,9 +585,7 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - release-windows-32bit:
-          requires:
-            - release-windows
+      - release-windows-32bit
```

Comment:
Probably need to add the filters then?

Reply:
No need. It didn't have any filters before.

Reply:
The dependency implicitly caused filtering?